omnibus/resources/chef-workstation/pkg/welcome.html.erb

```diff
@@ -12,7 +12,7 @@ Including:
 <ul>
   <li><strong>chef-run:</strong> Ad-hoc remote execution of Chef</li>
   <li><strong>ChefDK:</strong> Tools for local Chef Development</li>
```

Comment:
Though, we don't ship ChefDK anymore, do we?

Reply:
I didn't want to touch the issue of DK being bundled in this PR. That's written all over the place.

Reply:
Agreed, just wanted to bring it up 👍